app.js

```diff
@@ -4,38 +4,55 @@ const PORT = process.env.PORT || 3000;
 const Book = require('./api/models/bookDirectoryModel');
 const app = express();
 
-mongoose.connect('mongodb://localhost:27017/book', {
+mongoose.connect('mongodb+srv://cihat:cihat123.@realmcluster.kbrok.mongodb.net/test', {
```

Comment:
bunları .env'ye alacağız. https://www.npmjs.com/package/dotenv bu paket de node projelerinde env kullanabilmek için.